txpipe/twopoint_null_tests.py

```diff
@@ -412,8 +412,9 @@ def write_output_sacc(self, meta, d, sacc_file, text):
         S = sacc.Sacc()
 
         f = self.open_input("shear_photoz_stack")
-        z = f["n_of_z/source2d/z"][:]
-        Nz = f[f"n_of_z/source2d/bin_0"][:]
+        z = f["n_of_z/source/z"][:]
```

Comment:
we used the 2d values because that was the entire sample.  Is there a reason it shouldn't be the full sample?

If the issue is that the DES Y1 data doesn't have the 2D sample you could just add it to the data as the weighted sum of all the others.

Reply:
Ok yes I think that was the problem, and we were a little confused what source2d is. @jjeffe14 can you try to put source2d back and put in an n(z) that is the weighted sum of all the bins?

Reply:
Done!